src/platform/site-wide/mega-menu/tests/megaMenu.cypress.spec.js

```diff
@@ -0,0 +1,138 @@
+// Relative imports.
+import { mockUser } from '@@profile/tests/fixtures/users/user.js';
+
+const testMobileMenuSections = () => {
+  cy.get('[data-e2e-id="about-va-1"]').should('not.be.visible');
```

Comment:
If you'd like to use the `getByTestId` [0] method included with Cypress Testing Library, you can set the attribute as `data-testid`. I think you can configure what attribute it uses for that as well, but not sure if it will end up applying to other tests. [1]

[0] https://testing-library.com/docs/queries/bytestid
[1] https://testing-library.com/docs/dom-testing-library/api-configuration

Reply:
I didn't know about `getByTestId`, nice! Would that be faster for the tests or is it the same?

> Is this replacing the mega menu Nightwatch test? Would you want to remove that as well?

I was asked to add cypress e2e tests to the mega menu, so to be honest I didn't even know it already had nightwatch e2e tests! But I definitely feel more confident with these cypress mega menu e2e tests! 🙂 Should I remove the nightwatch test you think [here](https://github.com/department-of-veterans-affairs/vets-website/blob/40c24474e99f9a1114add503183773fe8bba9fef/src/platform/testing/e2e/megamenu/megamenu.js) and [here](https://github.com/department-of-veterans-affairs/vets-website/blob/feb8da863ac62c4380014ac8003a573d7c9ececd/src/platform/testing/e2e/megamenu/megamenu.e2e.spec.js)?

Reply:
I think a lot of us would be happy to see the old tests gone. I say go for it. 🚀

Reply:
Oh awesome! Done + updated! 💯

Reply:
> I didn't know about `getByTestId`, nice! Would that be faster for the tests or is it the same?

I think it's just a shortcut.

> A shortcut to `container.querySelector(`[data-testid="${yourId}"]`)` (and it also accepts a `TextMatch`).